packages/mongodb-memory-server-core/src/util/MongoBinaryDownload.ts

```diff
@@ -19,8 +19,20 @@ import { RequestOptions } from 'https';
 
 const log = debug('MongoMS:MongoBinaryDownload');
 
-const retryableStatusCodes = [503, 500];
-const retryableErrorCodes = ['ECONNRESET', 'ETIMEDOUT', 'ENOTFOUND', 'ECONNREFUSED'];
+const retryableStatusCodes = [429, 500, 503];
+
+const retryableErrorCodes = [
+  'ECONNRESET',
+  'ETIMEDOUT',
+  'ENOTFOUND',
+  'ECONNREFUSED',
+  'EPIPE',
+  'EHOSTUNREACH',
+  'EAI_AGAIN',
+  'ENETUNREACH',
+  'ECONNABORTED',
+  'aborted',
+];
```

Comment:
I think it would be a good idea to add some comments to non-self-explaining errors, for example what is `EAI_AGAIN`? (i did not even know about this error before looking it up)

Reply:
Yes there are lots of such error codes - https://nodejs.org/api/errors.html#nodejs-error-codes